application/features/Audio.py

```diff
@@ -31,6 +31,7 @@
 from .Connection import Connection
 from .. import app
 from ..utils import find_free_port, get_headers_dict_from_str, local_auth
+import application
```

Comment:
Similar to what we have done in the other modules (script files), can we log with a logger using the current module name? i.e., `logger = logging.getLogger(__name__)`